.gitignore

```diff
@@ -5,3 +5,4 @@ tsconfig.build.tsbuildinfo
 tsconfig.tsbuildinfo
 packages/**/dist
 .env
+*.feature
```

Comment:
Should be about a locally installed/cloned mockserver repo ?

Reply:
Yes, I don't want to see any cucumber files in this repo